File: src/main/java/com/codelog/lastlife/commands/CommandSetLives.java
package com.codelog.lastlife.commands;

import com.codelog.lastlife.LifeContext;
import org.bukkit.Bukkit;
import org.bukkit.ChatColor;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;
import org.jetbrains.annotations.NotNull;

public class CommandSetLives implements CommandExecutor {
    @Override
    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, @NotNull String[] args) {
        if (args.length != 2) {
            sender.sendMessage("Wrong number of arguments!");
            return false;
        }

        Player p;
        if ((p = Bukkit.getServer().getPlayer(args[0])) != null) {
            int lives;
            try {
                lives = Integer.parseInt(args[1]);
            } catch (NumberFormatException e) {
                sender.sendMessage(ChatColor.RED + "Invalid number of lives!");
                return true;
            }

            p.sendMessage(ChatColor.BLUE + "Your lives have been set to " + ChatColor.GREEN + lives);
            LifeContext.get().setPlayerLives(p.getUniqueId(), lives);

            sender.sendMessage(ChatColor.BLUE + "Player " + ChatColor.YELLOW + p.getName() + ChatColor.BLUE
                    + " has had their lives changed to " + lives);
        } else {
            sender.sendMessage(ChatColor.RED + "No such player is online!");
        }

        return true;
    }
}

File: src/main/kotlin/com/codelog/lastlife/commands/CommandSetLives.kt
package com.codelog.lastlife.commands

import com.codelog.lastlife.Command
import com.codelog.lastlife.CommandDataPackage
import com.codelog.lastlife.LifeContext
import net.kyori.adventure.text.Component
import net.kyori.adventure.text.format.TextColor
import org.bukkit.Bukkit
import org.bukkit.ChatColor
import org.bukkit.entity.Player

class CommandSetLives(override val label: String = "setlives") : Command() {
    override fun execute(dataPackage: CommandDataPackage): Boolean {
        if (dataPackage.args.size != 2) {
            dataPackage.sender.sendMessage(
                Component.text("Incorrect number of arguments!", TextColor.color(0xff0000))
            )
            return false
        }

        val p: Player? = Bukkit.getServer().getPlayer(dataPackage.args[0])
        if (p != null) {
            val lives: Int?
            try {
                lives = dataPackage.args[1].toInt()
            } catch (e: NumberFormatException) {
                dataPackage.sender.sendMessage(
                    Component.text("Invalid number of lives", TextColor.color(0xff0000))
                )
                return true
            }

            dataPackage.sender.sendMessage(ChatColor.RED.toString() + "You lives have been set to " + ChatColor.GREEN + lives)
            LifeContext.setPlayerLives(p.uniqueId, lives)
            dataPackage.sender.sendMessage(ChatColor.BLUE.toString() + "Player " + ChatColor.YELLOW + p.name + ChatColor.BLUE
                    + " has had their lives changed to " + lives
            )
        } else {
            dataPackage.sender.sendMessage(
                Component.text("No such player is online!", TextColor.color(0xff0000))
            )
        }

        return true
    }
}